File: AR-VR/thisiskartikgupta_KartikGupta_1923co1043_3/GettingStarted/isANumberPrime.java
import java.util.*;

public class isANumberPrime {
  public static void main(String[] args) {
    Scanner sc = new Scanner(System.in);
    int t = sc.nextInt();
    
    while(t-- != 0) {
        int n = sc.nextInt();
        boolean isPrime = true;
        for(int i = 2; i <= n/2; i++) {
            if(n % i == 0) {
                isPrime = false;
            }
        }
        
        if(isPrime)
          System.out.println("prime");
          
          else
              System.out.println("not prime");
    }
  }
}

File: ML/Saksham-Gupta-30_SakshamGupta_2024csit1164_2/GettingStarted/IsANumberPrime.cpp
#include <iostream>
using namespace std;

int isPrime(int n)
{
    int j;
    for (j = 2; j <= n/2; j++)
    {
        if (n % j == 0)
        {
            return 0;
        }
        else
        {
            continue;
        }
    }
    return 1;
}

int main()
{
    int t, i, n;
    cin >> t;
    for (i = 0; i < t; i++)
    {
        cin >> n;
        if (isPrime(n) == 1)
            cout << "prime" << endl;
        else
            cout << "not prime" << endl;
    }
}